schemas/2019-01-21/Microsoft.Kusto.json

```diff
@@ -231,18 +231,17 @@
     },
     "DatabaseProperties": {
       "properties": {
-        "softDeletePeriodInDays": {
-          "type": "integer",
-          "description": "The number of days data should be kept before it stops being accessible to queries."
+        "softDeletePeriod": {
+          "type": "string",
+          "format": "duration",

```

Comment:
>duration [](start = 21, length = 8)

Visual Studio doesn't support "duration" format today.. I guess we can remove it for now.